pepper-apis/src/main/java/org/broadinstitute/ddp/route/GetWorkflowRoute.java

```diff
@@ -72,11 +70,7 @@ public WorkflowResponse handle(Request request, Response response) {
             LOG.info("Finding suggestions using 'from' state {}", fromState);
 
             return workflowService
-                    .suggestNextState(handle, userGuid, studyGuid, fromState)
-                    .map(nextState -> {
-                        checkAndCreateStartActivityInstance(handle, fromState, nextState, operatorGuid, userGuid);
-                        return nextState;
-                    })
+                    .suggestNextState(handle, operatorGuid, userGuid, studyGuid, fromState)
```

Comment:
Since the functionality is now incorporated into the `WorkflowService`, there's no need for special handling anymore.